shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -114,6 +115,14 @@ - (instancetype)initWithName:(NSString*)labelPrefix
   else
     _dartProject.reset([project retain]);
 
+  if (!EnableTracingIfNecessary([_dartProject.get() settings])) {
+    NSLog(@"Cannot create a FlutterEngine instance in debug mode without Flutter tooling or Xcode. "
+          @"To create FlutterEngine instances from applications launched from the home-screen, use "
+          @"profile or release modes instead.");
+    [self release];
```

Comment:
`@To launch in debug mode, run flutter run from Flutter tools, run from an IDE with a Flutter IDE plugin or run the iOS project from Xcode`

It's implicit but we should describe the solution explicitly.

Reply:
Ack.